metadata/boundaries/school_district_boundaries.md

```diff
@@ -0,0 +1,77 @@
+# Title
+
+Utah School District Boundaries
+
+## ID
+
+fb17c1bb-954a-4a3b-997a-7024cae04fbc
+
+## Brief Summary
+
+Polygon dataset representing more than 40 school districts in Utah.
```

Comment:
**question**: should we add what types of schools since we have a few different layers?